examples/ensurance/evict-on-mem-usage-total/pod.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: low-pi6
+  labels:
+    k8s-app: low
+    preemptible_job: "true"
+spec:
+  containers:
+    - image: ccr.ccs.tencentyun.com/tkeimages/stress-ng:v0.12.09
+      command:
+        - stress-ng
+        - --vm-hang
+        - "3600"
+        - --vm
+        - "2"
+        - --vm-bytes
+        - "2G"
+      name: stress
+      volumeMounts:
+        - mountPath: /data
+          name: data
+  volumes:
+    - hostPath:
+        path: /data/dd
```

Comment:
Why we added a path to the container? It is necessary

Reply:
ok